src/style.css

```
@@ -0,0 +1,40 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--0 :nth-child(n) {
+  background-image: url("images/star.svg");
+}
+
+.stars--1 :nth-child(n+2) {
+  background-image: url("images/star.svg");
+}
+
+.stars--2 :nth-child(n+3) {
+  background-image: url("images/star.svg");
+}
+
+.stars--3 :nth-child(n+4) {
+  background-image: url("images/star.svg");
+}
+
+.stars--4 :nth-child(n+5) {
+  background-image: url("images/star.svg");
+}
+
+.stars--5 :nth-child(n+6) {
+  background-image: url("images/star.svg");
+}
```

combine this into one rule
like this

```suggestion
.stars--1 :nth-child(n+2),
.stars--2 :nth-child(n+3) {
  background-image: url("images/star.svg");
}
```